SDL/GoMePubNew.sdl

```diff
@@ -0,0 +1,76 @@
+STATEDESC GoMePubNew
+{
+    VERSION 1
+    
+#Game Variables
+    VAR BOOL        islmGZBeamVis[1]            DEFAULT=1   DEFAULTOPTION=VAULT
+
+}
+
+STATEDESC GoMePubNew
+{
+    VERSION 2
+    
+#Game Variables
+    VAR BOOL        islmGZBeamVis[1]            DEFAULT=0   DEFAULTOPTION=VAULT
+
+}
+
+STATEDESC GoMePubNew
+{
+    VERSION 3
+
+    VAR BOOL        islmGZBeamVis[1]            DEFAULT=0
+
+}
+
+STATEDESC GoMePubNew
+{
+    VERSION 4
+
+    VAR BOOL        BarDoorStat[1]              DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL        islmGZBeamVis[1]            DEFAULT=0   DEFAULTOPTION=VAULT
+
+}
+
+STATEDESC GoMePubNew
+{
+    VERSION 5
+
+    VAR BOOL        BarDoorStat[1]              DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL        islmGZBeamVis[1]            DEFAULT=0   DEFAULTOPTION=VAULT
+
+}
+
+STATEDESC GoMePubNew
+{
+    VERSION 6
+    
+    VAR BOOL        BarDoorStat[1]              DEFAULT=1   DEFAULTOPTION=VAULT
+    VAR BOOL        islmGZBeamVis[1]            DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL        SinkStat[1]                 DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL        ChisoBookVis[1]             DEFAULT=0   DEFAULTOPTION=VAULT
+    
+}
+
+STATEDESC GoMePubNew
+{
+    VERSION 7
+    
+    VAR BOOL        BarDoorStat[1]              DEFAULT=1
+    VAR BOOL        islmGZBeamVis[1]            DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL        SinkStat[1]                 DEFAULT=0
+    VAR BOOL        ChisoBookVis[1]             DEFAULT=0   DEFAULTOPTION=VAULT
+}
+
+STATEDESC GoMePubNew
+{
+    VERSION 8
```

Comment:
I'd really prefer we start at Version 1 for a new Age being added.  These files are prone to grow well enough without coming in from the start with development cruft. ;)